### DISCUSSION TITLE
How to assign name to a user when using email login ?

### DISCUSSION THREAD

**Discussion by WestFarmer** [hoppscotch/hoppscotch#3578] [Q&A] [answered]
![图片](https://github.com/hoppscotch/hoppscotch/assets/14016562/d19d7c39-4698-4213-8e67-4a460670fa5b)

please help.

**Comment by liyasthomas** [comment, marked as answer]
You can change your profile name from https://hoppscotch.io/profile if you're on Hoppscotch Cloud.
Or if you're a self-hosted instance, simply visit the `/profile` page to edit the account name.
